kbase-extension/static/kbase/css/kbaseNarrative.css

```diff
@@ -3079,6 +3082,69 @@ button.kb-data-obj {
     border: none;
 }
 
+/* style for Import As dropdown container */
+.kb-import-content .select2 .select2-selection--single {
+    width: 160px;
+    height: 42px;
+    background: #FFFFFF;
+    border: 1px solid #DF0002;
+    box-sizing: border-box;
+    border-radius: 4px;
+}
+
+/* When user clicks into the dropdown or chooses a selection box change border color */
+.kb-import-content .select2.select2-container--focus .select2-selection--single,
+.kb-import-content .select2.select2-container--open .select2-selection--single,
+.kb-import-content .select2-selection.select2-selection--single.type-selected {
+    border: 1px solid #C4C4C4;
+}
+
+/* style for Import As dropdown container text */
+.kb-import-content .select2.select2-container .select2-selection__rendered {
+    position: absolute;
+    left: 8.87%;
+    right: 5.41%;
+    top: 30.95%;
+    bottom: 30.95%;
```

Comment:
did these come out of figma or from your own calculations?

Reply:
these were all directly from figma